src/App.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import './App.css';
+import { getTodos, getUsers } from './api';
+import { ToDoList } from './components/ToDoList';
+import { Button } from 'semantic-ui-react';
+import { sortTodos } from './components/sortTodos';
+import { Users } from './components/Users';
+class  App extends React.Component {
+  state = {
+    todos: [],
+    users: [],
+    load: false,
+    buttonInnerText: 'Load Data',
+    disableButton: false
+  }
+  sortedList = (array,sortField) => {
+    this.setState({ todos: sortTodos(array, sortField) })
+  }
+  async loadInformation() {
+    this.setState({
+      disableButton: true,
+      buttonInnerText: "Loading...",
+    });
+    const [ todos, users ] = await Promise.all([
+      getTodos(),
+      getUsers(),
+    ]);
+    const fullArr = [ ...todos];
+    fullArr.map(el => el.user = users.find(user => user.id === el.userId));
```

Comment:
```suggestion
    fullArr.map(el => ({
      ...el,
      user: users.find(user => user.id === el.userId)
    }));
```

Reply:
when you use `map` you should return final result, not an assignment

Reply:
done